src/Sulu/Bundle/WebsiteBundle/Twig/Content/ContentTwigExtension.php

```diff
@@ -90,15 +90,48 @@ public function load($uuid)
                 $this->requestAnalyzer->getCurrentLocalization()->getLocale()
             );
 
-            return $this->structureResolver->resolve($contentStructure);
+            if ($properties === null) {
+                return $this->structureResolver->resolve($contentStructure);
```

Comment:
we could even deprecate calling this without a `properties` parameter. not sure if there are usecases for loading all extensions and properties

Reply:
Isn't this what the WebsiteController is dong calling the structureResolver without any options?

Reply:
The WebsiteController is calling ParameterResolver which does this: https://github.com/sulu/sulu/blob/466edcb9beb7debe5c64e75bade7d04bbb6164de/src/Sulu/Bundle/WebsiteBundle/Resolver/ParameterResolver.php#L83

Reply:
Yes - i moved the comment from the `StructureResolver` to the `ContentTwigExtension` 🙂  Would only deprecate it in the `ContentTwigExtension`

Reply:
yeah deprecating it there sounds good for me.